boards/arduino-mega2560/board.c

```diff
@@ -56,7 +56,7 @@ void board_init(void)
 void SystemInit(void)
 {
     /* initialize UART_0 for use as stdout */
```

Comment:
Yes if you have time please open an issue to fix it, it was already discussed [here](https://github.com/RIOT-OS/RIOT/issues/6264#issuecomment-269384915) but we forgot to do the renaming.

Thanks!